### ISSUE TITLE
Issue template is not working anymore

### ISSUE THREAD

**Issue by stdedos** [koalaman/shellcheck#3241]
https://github.com/koalaman/shellcheck/blob/master/.github/ISSUE_TEMPLATE.md

<img width="893" height="918" alt="Image" src="https://github.com/user-attachments/assets/93639d96-b265-402a-9ade-778bfd620bc0" />

I'm happy to PR-fix this IF-F there is some @koalaman activity :v:

-----

PS: Workaround: Copy from here:

````md
#### For bugs
- Rule Id (if any, e.g. SC1000):
- My shellcheck version (`shellcheck --version` or "online"):
- [ ] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [ ] I tried on https://www.shellcheck.net/ and verified that this is still a problem on the latest commit

#### For new checks and feature suggestions
- [ ] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/sh
your script here
```

#### Here's what shellcheck currently says:



#### Here's what I wanted or expected to see:

````

which is _almost a copy_ from here https://raw.githubusercontent.com/koalaman/shellcheck/refs/heads/master/.github/ISSUE_TEMPLATE.md

**Comment by koalaman** [comment]
I used the GitHub wizard to set up these templates and it appears to have worked. Thanks!

**Comment by stdedos** [comment]
Happy to see you back & active 😄

**Comment by e-kwsm** [comment]
https://github.com/koalaman/shellcheck/blob/master/.github/ISSUE_TEMPLATE.md is obsolescent and can be removed?

**Comment by stdedos** [comment]
btw @koalaman you haven't made your templates exclusive.

i.e. "There are ways" for people to ignore them.

As you have a LOT of issues, I'd say "there isn't much" that doesn't fit your templates - and you could disallow creating a blank issue.
(If you are afraid that blocking blank issues as an issue, maybe perhaps open Discussions?)
